sdk/purview/purview-scanning-rest/test/public/dataSources.spec.ts

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
-import type { PurviewScanningRestClient, DataSource } from "../../src";
+import type { PurviewScanningClient, DataSource, DataSourceListOutput } from "../../src";
```

Comment:
should we use title to set the client name as before?

Reply:
if we change the title, the client file will be rename